lib/board.rb

```diff
@@ -0,0 +1,45 @@
+class Board
+  attr_accessor :cells
+
+  @cells = []
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(input)
+    @cells[input.to_i-1]
+  end
+
+  def full?
+    @cells.include?(" ") ? false : true
```

Comment:
There are a couple ways to solve this that don't involve comparing the expression from the include? method to a boolean value. 

The first is 
```ruby
!@cells.include?(" ")
```

The next involves calling the .all? method on @cells.

`cells.all? {|cell| cell != " "}`